airflow/docker.go

```diff
@@ -301,10 +299,23 @@ func (d *DockerCompose) Start(imageName, settingsFile, composeFile, buildSecretS
 		startupTimeout = 5 * time.Minute
 	}
 
-	err = checkWebserverHealth(settingsFile, envConns, project, d.composeService, airflowDockerVersion, noBrowser, startupTimeout)
+	// Airflow webserver should be hosted at localhost
+	// from the perspective of the CLI running on the host machine.
+	healthUrl := "http://localhost:8080/health"
+
+	// Check the health of the webserver, up to the timeout.
+	// If we fail to get a 200 status code, we'll return an error message.
+	err = checkWebserverHealth(healthUrl, startupTimeout)
+	if err != nil {
+		return err
+	}
+
+	// If we've successfully gotten a healthcheck response, print the status.
+	err = printStatus(settingsFile, envConns, project, d.composeService, airflowDockerVersion, noBrowser)
```

Comment:
`printStatus()` was previously called by the original `checkWebserverHealth()` function. That function has been removed and replaced and doesn't concern itself with printing the message. That now just happens at the "top-level" here.

Reply:
Good call.